librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -406,6 +406,8 @@ void QC_ApplicationWindow::loadPlugins() {
                 continue;
             #endif
 
+            qDebug() << fileName;
+
```

Comment:
we use `RS_DEBUG->print()` instead of qDebug()
also all added debug output in this file seems from testing and should be removed before merging